test/src/test/java/aima/test/unit/search/basic/BasicNodeTest.java

```diff
@@ -0,0 +1,38 @@
+package aima.test.unit.search.basic;
```

Comment:
No need for this to be part of a BidirectionalSearch pull request. This is a general test but won't be adding as it is based on JSON toString() output expectations.